### PR TITLE
feat: SAP recommendations added

### PR DESCRIPTION
**Pull request by humblejay**
# Overview/Summary

Added SAP Netapp files & Storage related recommendations

## Related Issues/Work Items

AVD ADO 36151

### Breaking Changes

None

## As part of this pull request I have

- [x] Read the [Contribution Guide](https://azure.github.io/Azure-Proactive-Resiliency-Library-v2/contributing) and ensured this PR is compliant with the guide
- [x] Checked for duplicate [Pull Requests](https://github.com/Azure/Azure-Proactive-Resiliency-Library-v2/pulls)
- [x] Associated it with relevant [GitHub Issues](https://github.com/Azure/Azure-Proactive-Resiliency-Library-v2/issues) or ADO Work Items (Internal Only)
- [x] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Azure-Proactive-Resiliency-Library-v2/tree/main)
- [x] Ensured PR tests are passing
- [x] Performed testing and provided evidence (e.g. screenshot of output) for any changes associated to ARG queries
- [x] Updated relevant and associated documentation (e.g. Contribution Guide, Docs etc.)

### CODE REVIEW

File: azure-specialized-workloads/sap/kql/.kql
Comment (ejhenry) [inline azure-specialized-workloads/sap/kql/.kql:1]: @humblejay was the addition of this file unintentional? If so, please remove from the PR

**Comment by ejhenry** [comment]
@Azure/aprl-sap please review

File: azure-specialized-workloads/sap/kql/.kql
Comment (humblejay) [reply azure-specialized-workloads/sap/kql/.kql:1]: removed, that was unintentional

**Review by PmeshramPM — COMMENTED** [review]
Hi @humblejay - On the links for ANF , please use these ones. These links have the Cluster parameter when you are using ANF for SUSE or RHEL clusters. 

https://learn.microsoft.com/en-us/azure/sap/workloads/high-availability-guide-suse-nfs-simple-mount?tabs=lb-portal%2Censa1

https://learn.microsoft.com/en-us/azure/sap/workloads/high-availability-guide-suse-nfs-simple-mount?tabs=lb-portal%2Censa1

https://learn.microsoft.com/en-us/azure/sap/workloads/high-availability-guide-suse-netapp-files?tabs=lb-portal%2Censa1


https://learn.microsoft.com/en-us/azure/sap/workloads/high-availability-guide-suse-netapp-files?tabs=lb-portal%2Censa1

https://learn.microsoft.com/en-us/azure/sap/workloads/high-availability-guide-rhel-netapp-files?tabs=lb-portal%2Censa1

https://learn.microsoft.com/en-us/azure/sap/workloads/sap-hana-high-availability-netapp-files-red-hat?tabs=lb-portal

https://learn.microsoft.com/en-us/azure/sap/workloads/sap-hana-high-availability-netapp-files-suse?tabs=lb-portal

**Comment by ejhenry** [comment]
@Azure/aprl-sap please review

**Review by oZakari — CHANGES_REQUESTED** [review]

**Review by PmeshramPM — APPROVED** [review]
approved

**Pull request merged**